_sass/components/_blockquote-section.scss

```diff
@@ -0,0 +1,109 @@
+$bp-quote-sm: 62.5rem;
+$bp-quote-md: 86.25rem;
+$bp-quote-lg: 106.25rem;
+$bp-quote-xl: 125rem;
+
+.c-blockquote {
+  width: 100%;
+  padding: 6rem 0;
+  margin: 4rem auto;
+  background: #F4F4F4;
+
+  &__inner-container {
+    width: 80%;
+    margin: 0 auto;
+    position: relative;
+  }
+
+  &__image-container {
+    position: relative;
+    width: 100%;
+    height: 100%;
+  }
+
+  &__image-container {
```

Comment:
This is the second place you're declaring `&__image-container`. We can combine these into one so that `&__image-container` only shows up once in the file.

Reply:
oops